ucsb-cs156-s24/proj-happycows-s24-4pm-5
# Fix: homepage common card background issue

## Overview

In this PR, I fix the background color issue on the homepage by changing back to the color array. Also, I change the text font for common name and the buttons

## Screenshots (Optional)

<img width="1501" alt="Screenshot 2024-05-28 at 5 38 50 PM" src="https://github.com/ucsb-cs156-s24/proj-happycows-s24-4pm-5/assets/97069739/01f4b848-8057-4b6c-921d-a4fc60b09f81">


## Validation (Optional)

1. Go to https://happycows-congruidu-dev.dokku-05.cs.ucsb.edu/
2. Go to homepage
4. See if the common card background color is going to change after refreshing the page, and the text fonts are different than before

## Tests


## Linked Issues

Closes https://github.com/ucsb-cs156-s24/proj-happycows-s24-4pm-5/issues/66
